rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -289,6 +289,12 @@ class Publisher : public PublisherBase
     }
   }
 
+  void
+  publish(std::shared_ptr<rcl_serialized_message_t> serialized_msg)
```

Comment:
Should the parameter be const to avoid modification?

Reply:
done in https://github.com/ros2/rclcpp/pull/541/commits/7e5b4168b402f50b576fe7d94d2e82cb8adb7479